test/sql/aggregate/aggregates/test_state_export.test

```diff
@@ -204,3 +204,35 @@ query IIIIIII nosort res10
 SELECT g, finalize(count_star_state),finalize(count_state), finalize(sum_state), finalize(avg_state)::integer, finalize(min_state), finalize(max_state) FROM state_view ORDER BY g;
 ----
 
+# BLOB casting back and forth
+statement ok
+SELECT (SUM(42) EXPORT_STATE)::BLOB;
+
+query I
+SELECT FINALIZE(COMBINE(SUM(42) EXPORT_STATE, (SUM(42) EXPORT_STATE)::BLOB));
+----
+84
+
+statement error
+SELECT (SUM(42) EXPORT_STATE)::INTEGER;
+
+
+# simulate round tripping
+require parquet
+
+statement ok
+COPY (SELECT g, (SUM(d) EXPORT_STATE)::BLOB s1 FROM dummy GROUP BY g) TO '__TEST_DIR__/state.parquet' (FORMAT PARQUET);
+
+query II
+SELECT g, FINALIZE(COMBINE(s2, s1)) FROM (SELECT g, SUM(d) EXPORT_STATE s2 FROM dummy GROUP BY g) q1 JOIN '__TEST_DIR__/state.parquet' USING(g) ORDER BY g;
+----
+0	900
+1	920
+2	940
+3	960
+4	980
+5	1000
+6	1020
+7	1040
+8	1060
+9	1080
```

Comment:
Perhaps some binding error tests?

Reply:
Good point found two asserts that should have been exceptions.